Description: Ensure that DGEMM regression test is not run
 It can't be run because it depends on LAPACKE, which we do not build from
 OpenBLAS.
Origin: upstream, https://github.com/xianyi/OpenBLAS/commit/a1fce677435a79d3cb577086793556d87ff76552
Bug: https://github.com/xianyi/OpenBLAS/issues/2215
Last-Update: 2019-08-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/utest/CMakeLists.txt
+++ b/utest/CMakeLists.txt
@@ -38,9 +38,14 @@ if (NOT NO_LAPACK)
 set(OpenBLAS_utest_src
   ${OpenBLAS_utest_src}
   test_potrs.c
+  )
+if (NOT NO_CBLAS AND NOT NO_LAPACKE)
+set(OpenBLAS_utest_src
+  ${OpenBLAS_utest_src}
   test_kernel_regress.c
   )
 endif()
+endif()
 
 set(OpenBLAS_utest_bin openblas_utest)
 add_executable(${OpenBLAS_utest_bin} ${OpenBLAS_utest_src})
--- a/utest/Makefile
+++ b/utest/Makefile
@@ -4,6 +4,9 @@ TOPDIR	= ..
 override TARGET_ARCH=
 override TARGET_MACH=
 
+override TARGET_ARCH=
+override TARGET_MACH=
+
 UTESTBIN=openblas_utest
 
 .PHONY : all
@@ -16,8 +19,12 @@ OBJS=utest_main.o test_amax.o test_rotmg
 
 ifneq ($(NO_LAPACK), 1)
 OBJS += test_potrs.o
+ifneq ($(NO_CBLAS), 1)
+ifneq ($(NO_LAPACKE), 1)
 OBJS += test_kernel_regress.o
 endif
+endif
+endif
 
 #this does not work with OpenMP nor with native Windows or Android threads
 # FIXME TBD if this works on OSX, SunOS, POWER and zarch
